README.md

```diff
@@ -57,11 +57,11 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [Math__magicians] <a name="about-project"></a>
+
+> This project is a Book Store app I created with react js👌, the user can add any book by the app. 
```

Comment:
- [ ] The name of the project is _**Bookstore**_ and not **_Math Magician_**. Kindly change the name. 👍